library/src/scripts/flyouts/FlyoutToggle.tsx

```diff
@@ -4,18 +4,17 @@
  * @license GPL-2.0-only
  */
 
-import React, { useRef, useMemo, useState, useCallback, useEffect, useLayoutEffect } from "react";
-import FocusWatcher, { useFocusWatcher } from "@library/dom/FocusWatcher";
-import EscapeListener, { useEscapeListener } from "@library/dom/EscapeListener";
-import { t } from "@library/utility/appUtils";
-import { ButtonTypes } from "@library/forms/buttonStyles";
-import { getRequiredID, uniqueIDFromPrefix } from "@library/utility/idUtils";
-import Button from "@library/forms/Button";
 import { dropDownClasses } from "@library/flyouts/dropDownStyles";
-import classNames from "classnames";
+import Button from "@library/forms/Button";
+import { ButtonTypes } from "@library/forms/buttonStyles";
 import Modal from "@library/modal/Modal";
 import ModalSizes from "@library/modal/ModalSizes";
+import { t } from "@library/utility/appUtils";
+import { uniqueIDFromPrefix } from "@library/utility/idUtils";
+import classNames from "classnames";
+import React, { useCallback, useEffect, useMemo, useRef, useState } from "react";
 import { forceRenderStyles } from "typestyle";
+import { useFocusWatcher, useEscapeListener } from "@vanilla/react-utils";
```

Comment:
Is there any way to commit only real changes but not reordered lines?
In this case 2 lines where deleted:
```
import FocusWatcher, { useFocusWatcher } from "@library/dom/FocusWatcher";
import EscapeListener, { useEscapeListener } from "@library/dom/EscapeListener";
```
and replaced with one new line:
```
import { useFocusWatcher, useEscapeListener } from "@vanilla/react-utils";
```
That would help to review and track repo changes.

Reply:
This type of thing is mostly the cause of using the typescript refactoring tool "Cleanup Imports". It rewrites all the import declarations, sorted, and with no unused values.

Personally I don't think the imports are of much concern for a review. If the are incorrect the build will fail.

Reply:
>  If the are incorrect the build will fail.
Completely disagree to this argument. :-)
Do we need a review at all if the build does not fail?

Reply:
It's not garbage. I trust the import generation of the typescript language. I also trust that broken imports break the build. If the file does not exist, it will fail loudly and plainly.